src/Microsoft.Framework.CopyOnWriteDictionary.Internal/CopyOnWriteDictionary.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Framework.Internal
```

Comment:
other than namespaces, have you made any changes to *.cs files in src directories?